simulator/src/app.js

```
@@ -1,6 +1,15 @@
+import * as Sentry from "@sentry/browser";
 import { setup } from './setup';
 import Array from './arrayHelpers';
```

Biome

[error] Do not shadow the global "Array" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)